tools/pelican/content/ipynb/germany-compute-hospitalisierungsrate.ipynb

```diff
@@ -0,0 +1,4575 @@
+{
```

Comment:
<p>I would either uncomment or remove this part.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oscovida/oscovida/pull/282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I have left this in the notebook, but added an explanation at the top of the notebook, why this is in there.</p>